Description: Adapt some files to Debian packaging:
 - /usr/bin/solr
 - /etc/default/solr.in.sh
Author: Didier Raboud <odyx@liip.ch>
Origin: vendor

--- a/bin/solr
+++ b/bin/solr
@@ -70,8 +70,10 @@
   fi
 done
 
-SOLR_TIP=`dirname "$SOLR_SCRIPT"`/..
-SOLR_TIP=`cd "$SOLR_TIP"; pwd`
+# Debian package directories
+SOLR_TIP="/usr/share/solr"
+SOLR_LOGS_DIR="/var/log/solr"
+
 DEFAULT_SERVER_DIR="$SOLR_TIP/server"
 
 # If an include wasn't specified in the environment, then search for one...
--- a/bin/solr.in.sh
+++ b/bin/solr.in.sh
@@ -69,7 +69,7 @@
 # RMI_PORT=18983
 
 # Set the thread stack size
-SOLR_OPTS="$SOLR_OPTS -Xss256k"
+SOLR_OPTS="-Xss256k"
 
 # Anything you add to the SOLR_OPTS variable will be included in the java
 # start command line as-is, in ADDITION to other options. If you specify the
@@ -80,20 +80,20 @@
 
 # Location where the bin/solr script will save PID files for running instances
 # If not set, the script will create PID files in $SOLR_TIP/bin
-#SOLR_PID_DIR=
+SOLR_PID_DIR="/var/lib/solr"
 
 # Path to a directory for Solr to store cores and their data. By default, Solr will use server/solr
 # If solr.xml is not stored in ZooKeeper, this directory needs to contain solr.xml
-#SOLR_HOME=
+SOLR_HOME="/var/lib/solr/data"
 
 # Solr provides a default Log4J configuration properties file in server/resources
 # however, you may want to customize the log settings and file appender location
 # so you can point the script to use a different log4j.properties file
-#LOG4J_PROPS=/var/solr/log4j.properties
+LOG4J_PROPS="/var/lib/solr/log4j.properties"
 
 # Location where Solr should write logs to; should agree with the file appender
 # settings in server/resources/log4j.properties
-#SOLR_LOGS_DIR=
+SOLR_LOGS_DIR="/var/log/solr"
 
 # Sets the port Solr binds to, default is 8983
 #SOLR_PORT=8983
